packages/checkbox-group/src/LionCheckboxIndeterminate.js

```diff
@@ -0,0 +1,110 @@
+import { html, css } from '@lion/core';
+import { LionCheckbox } from './LionCheckbox.js';
+
+export class LionCheckboxIndeterminate extends LionCheckbox {
+  static get properties() {
+    return {
+      /**
+       * Indeterminate state of the checkbox
+       */
+      indeterminate: {
+        type: Boolean,
+        reflect: true,
+      },
+    };
+  }
+
+  get _checkboxGroupNode() {
+    return /** @type {import('./LionCheckboxGroup').LionCheckboxGroup} */ (this.parentElement);
```

Comment:
A safer choice would be `this.__parentFormGroup`. In this way, we can allow users more freedom in terms of html (adding a div, an ul/li etc.)